jekyll/_cci2/using-gpu.md

```diff
@@ -4,38 +4,81 @@ title: "Using the GPU execution environment"
 description: "Learn how to configure a your jobs to run in the GPU execution environment."
 version:
 - Cloud
-- Server v3.x
-- Server v2.x
+plan:
+- Scale
 ---
-[server-gpu]: {{ site.baseurl }}/2.0/gpu/
 
-CircleCI Cloud has execution environments with Nvidia GPUs for specialized workloads. The hardware is Nvidia Tesla T4 Tensor Core GPU, and our GPU executors come in both Linux and Windows VMs.
+You can run your jobs in the GPU execution environment, using either Windows or Linux virtual machines, for access to Nvidia GPUs for specialized workloads. 
+
+To use the Linux GPU execution environment, use the machine executor and specify a GPU-enabled image. For a full list of machine executor images see the [CircleCI Developer Hub](https://circleci.com/developer/images?imageType=machine) or the [Configuration Reference]({{site.baseurl}}/2.0/configuration-reference#available-linux-gpu-images).
 
-{:.tab.gpublock.Linux}
 ```yaml
 version: 2.1
 
 jobs:
   build:
     machine:
-      resource_class: gpu.nvidia.small
-      image: ubuntu-1604-cuda-10.1:201909-23
+      image: ubuntu-2004-cuda-11.4:202110-01
     steps:
       - run: nvidia-smi
 ```
 
-{:.tab.gpublock.Windows}
+To use the Windows GPU execution environment, you can either choose to use the windows orb and specify the built-in GPU executor, or use the machine executor and specify a Windows GPU-enabled image. Refer to the [Orb Registry page](https://circleci.com/developer/orbs/orb/circleci/windows) for full details, and the [Developer Hub](https://circleci.com/developer/images?imageType=machine) for full details of available machine executor images.
+
+{:.tab.gpublock.Windows_GPU_with_orb}
 ```yaml
 version: 2.1
 
 orbs:
-  win: circleci/windows@2.3.0
+  win: circleci/windows@4.1.0
+
+jobs:
+  build:
+    executor: win/server-2019-cuda
+    steps:
+      - run: '&"C:\Program Files\NVIDIA Corporation\NVSMI\nvidia-smi.exe"'
+```
+
+{:.tab.gpublock.Windows_GPU_with_machine}
+```yaml
+version: 2.1
 
 jobs:
   build:
-    executor: win/gpu-nvidia
+    machine: 
+      image: windows-server-2019-nvidia:stable
     steps:
       - run: '&"C:\Program Files\NVIDIA Corporation\NVSMI\nvidia-smi.exe"'
 ```
 
+## Available resource classes
+{: #available-resource-classes }
+
+Specify a resource class to fit your project and requirements. For further details on credit usage for these options, see the [Resource Class pricing and plans page](https://circleci.com/product/features/resource-classes/).
+
+### Linux GPU
+{: linux-gpu}
+
+```yaml
+version: 2.1
+
+jobs:
+  build:
+    machine:
+      image: ubuntu-2004-cuda-11.4:202110-01
+    resource_class: gpu.nvidia.small
+    steps:
+      - run: nvidia-smi
+```
+
+{% include snippets/gpu-linux-resource-table.md %}
+
+### Windows GPU
+{: windows-gpu}
+
+For Windows there is currently one resource class option.
```

Comment:
Is this true? We list Medium, Large, X Large and 2X Large on outer.

Reply:
Hey! Yes so that's for Windows VMs but if you look at Windows GPU there's just medium: https://circleci.com/pricing/